src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -104,7 +111,29 @@ function SpatialSymbolicPermutation(
         )
         valid = Base.Generator(idxs -> CartesianIndex{D}(idxs), ranges)
     end
-    SpatialSymbolicPermutation{D, p, typeof(valid)}(stencil, copy(stencil), arraysize, valid)
+    SpatialSymbolicPermutation{D, p, typeof(valid)}(stencil, copy(stencil), arraysize, valid, get_m(stencil))
+end
+
+# get stencil in the form of vectors of cartesian indices from either input type
+stencil_to_offsets(stencil::Vector{CartesianIndex{D}}) where D = stencil, D
+
+function stencil_to_offsets(stencil::NTuple{2, NTuple{D, T}}) where {D, T}
```

Comment:
I am confused about this git diff... Havent you already added these methods in PR #121 ?

Reply:
Yes, I think it has to do with me locally merging some upstream branch after committing my stuff (?)